Wiki.md

```diff
@@ -0,0 +1,108 @@
+# Commission and control a Matter Thread device via the OTBR Snap
+
+### Setup environment:
+Ubuntu Desktop 23.04
+
+### Requirements:
+
+nRF52840 dongle A with the Thread lighting app running on it
+
+nRF52840 dongle B with RCP firmware as an RCP
+
+## Installation and setup of OTBR Snap:
+1. Install the OpenThread Border Router (OTBR) snap:
+```bash
+snap install openthread-border-router
+```
+
+2. Configure the snap by updating the following settings based on your system:
+```bash
+$ sudo snap get openthread-border-router
+Key        Value
+infra-if   eno1
+radio-url  spinel+hdlc+uart:///dev/ttyACM0
+thread-if  wpan0
+
+$ sudo snap set openthread-border-router infra-if=eno1
+```
+
+3. Start the OTBR snap:
+```bash
+snap start openthread-border-router
+```
+
+## Forming the OTBR Network:
+1. Connect the RCP Dongle to a USB port
+2. Use the CTL tool to initialize the Thread network:
+```bash
+sudo openthread-border-router.ot-ctl
+> dataset init new
+Done
+> dataset commit active
+Done
+> ifconfig up
+Done
+> thread start
+Done
+```
+Alternatively, these steps could also be performed in the GUI at http://0.0.0.0/. 
```

Comment:
The loopback interface address should be localhost or 127.0.0.1. Adresse 0.0.0.0 may work on some systems but it isn't usually used for this purpose.